Jellyfin.Api/Controllers/StartupController.cs

```diff
@@ -32,20 +33,24 @@ public StartupController(IServerConfigurationManager config, IUserManager userMa
         /// <summary>
         /// Api endpoint for completing the startup wizard.
         /// </summary>
+        /// <returns>Status.</returns>
         [HttpPost("Complete")]
-        public void CompleteWizard()
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public ActionResult CompleteWizard()
         {
             _config.Configuration.IsStartupWizardCompleted = true;
             _config.SetOptimalValues();
             _config.SaveConfiguration();
+            return Ok();
```

Comment:
Is this required?

Reply:
It's not required, but in #2872 we decided to explicitly return status instead of void.

Reply:
@cvium In case you missed it, I collected all the conventions we decided on through various conversations here: https://github.com/jellyfin/jellyfin/issues/2872#issuecomment-617845103